test/integration/targets/vmware_content_library/aliases

```diff
@@ -0,0 +1,2 @@
+cloud/vcenter
```

Comment:
I want place all content library test files in vmware_content_library  folder. since prechecks were failing due to missing aliases I created one here. There is one more aliases in another PR which I raised for content library info module

Reply:
Ok. Make sense.